lib/clients/AwsClient.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, operation_name, configuration, directory_persistent, director
         }
         self.max_retries = (configuration.get('max_retries') if
                             type(configuration.get('max_retries'))
-                            == int else 10)
+                            == int else 15)
```

Comment:
Just a question - since the retries happen exponentially, what is the time lapse from the first attempt to the last attempt ? i.e. 15th attempt. Asking this Q as we want to just ensure the last attempt does not happen at a window by when the next run would have anyways run.